po/LINGUAS

```diff
@@ -0,0 +1 @@
+de
```

Comment:
I'd really prefer to not introduce this. We fully automate translation workflows, and this is a stupid redundant file. `man msgfmt` does not even mention it. If it can be replaced with command line arguments (as your commit message suggests), let's do that. Otherwise, let's generate the file on the FLY, let `make clean` remove it, and add it to .gitignore.

Reply:
> man msgfmt does not even mention it.

Remember gettext is a GNU software, so man pages are always second class citizens compared to e.g. info pages (sigh); this is described in the up-to-date online documentation though: 
https://www.gnu.org/software/gettext/manual/html_node/msgfmt-Invocation.html#XML-mode-operations

> We fully automate translation workflows,

FYI, translations systems such as Weblate have built-in capabilties for keeping these `LINGUAS` files updated; for example, Weblate has an addon for it:
https://docs.weblate.org/en/latest/admin/addons.html#update-linguas-file

I can imagine it is not ideal to keep a static file like that, partially redundant b/c of the .po files themselves; IME having it static is the least of the issue, and the maintenance overhead this costs is (again, IMHO) less than trying to make it automated. Also this file changes very rarely, only when a language is added or removed.

Reply:
I'm fine with writing the dynamic LINGUAS generation. In fact, it's not hard:

```
	(cd po; ls *.po | sed 's/\.po//') > po/LINGUAS
	msgfmt --xml -d po [...]
	rm -f po/LINGUAS
```